store/storetest/plugin_store.go

```diff
@@ -97,14 +97,14 @@ func doTestPluginSaveOrUpdate(t *testing.T, ss store.Store, s SqlSupplier, doer
 			ExpireAt: expireAt,
 		}
 
-		retKV, err := ss.Plugin().SaveOrUpdate(kv)
+		retKV, err := doer(kv)
```

Comment:
Wouldn't have expected `SaveOrUpdate` to return the KV struct back to the caller as store doesn't touch the struct, not sure what the rationale behind it is 🤔